contracts/core/RebalancingSetTokenFactory.sol

```diff
@@ -34,32 +34,45 @@ contract RebalancingSetTokenFactory {
 
     /* ============ State Variables ============ */
 
-    // Address of the Core contract
+    // Address of the Core contract used to verify factory when creating a Set
     address public core;
 
+    // Minimum amount of time between rebalances in seconds
+    uint256 public minimumRebalanceInterval;
+
+    // Minimum amount of time users can review proposals
```

Comment:
Nit: specify that this is in seconds as well

Reply:
+1